src/index.ts

```diff
@@ -0,0 +1,4 @@
+import FormField from "./Form/Field/FormField";
```

Comment:
I think folder names should be in lowercase, and could be separated with a dash if there is more than one word in the folder name: `form/field/FormField`, `input/Input`.

Should we move `input` inside `form` folder ?